doc/manual/manual.tex

```diff
@@ -7195,43 +7262,83 @@ \subsubsection{3D convection with an Earth-like initial condition}
-ridges are elevated due to (deep and) shallow upward flow. The bottom row
-shows the heat flux density at the surface for 2 refinement steps (e, colorbar
-ranges from 13 to 19 mW/$m^2$) and for 4 refinement steps (f, colorbar ranges
-from 35 to 95 mW/$m^2$). Similar to the dynamic topography, a first order correlation
+Figure~\ref{fig:ic-1}c and d shows the heat flux density at the surface for 
+2 refinement steps (c, colorbar ranges from 13 to 19 mW/$m^2$) and for 
+4 refinement steps (d, colorbar ranges from 35 to 95 mW/$m^2$). A first order correlation
 with upper mantle features such as high heat flow at mid ocean ridges and low
 heat flow at cratons is correctly initialized by the tomography model. 
+The mantle flow and bouyancy variations produce dynamic topography on the 
+top and bottomw surface, which is shown for 2 refinement steps (\ref{fig:ic-2}a and c, respectively) 
+and 4 refinement steps (\ref{fig:ic-2}b and d, respectively). 
+One can see that subduction zones are visible as depressed surface topography 
+due to the downward flow, while regions such as Iceland, Hawaii, or mid ocean 
+ridges are elevated due to (deep and) shallow upward flow. The core mantle boundary
+topography shows that the upwelling large low shear wave velocity 
+provinces deflect the core mantle boundary up. Lastly, Figure~\ref{fig:ic-2}e and 
+f show geoid perturbations for 2 and 4 global refinement steps, respectively. 
+The geoid anomalies show a strong correlation with the surface dynamic topography. 
+This is in part expected given that the geoid anomalies are driven by the 
+deflection of the upper and lower surface
+as well as internal density variations. The relative importance of these different
+contributors is dictated by the Earth's viscosity profile. Due to the isoviscous
+assumption in this cookbook, we don't properly recover patterns of the observed
+geoid. 
+
+As discussed in the previous cookbook, dynamic topography does not necessarily 
+average to zero if the resolution is not high enough. While one can simply subtract
+the mean as a postprocessing step this should be done with caution since a non-zero
+mean indicates that the refinement is not sufficiently high to resolve the
+convective flow. In Figure~\ref{fig:ic-2}a-d we refrained from subtracting the mean but 
+indicated it at the bottom left of each panel. The mean dynamic 
+topography approaches zero for increasing refinement. Furthermore, the mean bottom 
+dynamic topography is closer to zero than the mean top dynamic topography. This is 
+likely due to the larger magnitude of dynamic topography at the surface and the
+difference in resolution between the top and bottom domain
+(for a given refinement, the resolution at the core mantle boundary is
+higher than the resolution at the surface). The geoid postprocessor removes the
+average top and bottom surface when evaluating its contribution
+to the gravity field. The average geoid height is therefore zero. 
```

Comment:
Is this still the case now that it uses the new dynamic topography postprocessor?